releasetasks/util.py

```diff
@@ -0,0 +1,22 @@
+import os
+from taskcluster.utils import encryptEnvVar
+
+
+def treeherder_platform(platform):
+    # See https://github.com/mozilla/treeherder/blob/master/ui/js/values.js
+    m = {
+        "linux": "linux32",
+        "linux64": "linux64",
+        "macosx64": "osx-10-10",
+        "win32": "windowsxp",
+        "win64": "windows8-64",
+    }
+    return m[platform]
```

Comment:
Seems like this might be better off living in thclient - can you see what those folks think?